ansible/cassandra-trigger-deploy.yml

```diff
@@ -8,6 +8,12 @@
         dest: /etc/cassandra/triggers
       with_fileglob:
            - static-files/cassandra.transaction-event-handler-*.jar
+
+    - name: copy logback.xml file to cassandra trigger path
+      template:
+        src: "roles/cassandra-db-update/templates/logback.j2"
```

Comment:
Reference to the source file can be just the name of the file. We dont need to specify the full path of the file
src: logback.j2

Reply:
I added a new role to cassandra-triggers and added logback.j2 file there

Reply:
cassandra-trigger-deploy.yml is the main playbook. Please refer the cassandra-db-update role in this file and mention the template copy  steps in cassandra-db-update/tasks/main.yml